core/include/detray/propagator/navigator.hpp

```diff
@@ -565,6 +566,10 @@ class navigator {
         if (navigation.trust_level() == navigation::trust_level::e_full) {
             return navigation._heartbeat;
         }
+        if (navigation.trust_level() == navigation::trust_level::e_high) {
```

Comment:
This should be removed. After a navigation update the navigation must be at full trust or we should run through the diagnostics below